### PR TITLE
Docs docs baby

### PR DESCRIPTION
**Pull request by pataruco**
# Update docs

### CODE REVIEW

**Review by anubha22 — APPROVED** [review]
Makes sense to me!

File: readme.md [hunk 1]
```diff
@@ -1,25 +1,84 @@
...
-## What is it
+This GitHub Action will automatically gather compliance information about:
 
-GitHub Action that gather compliance evidence and store it in the cloud
+- Documentation
+- Test results
+- GitHub pull request information and commits associated
 
-## Inputs
+And stored in a Google Cloud Storage bucket for future reference for regulators.
```
Comment (DimplePatel) [inline readme.md:9]: Might be clearer to say 
"And store it in a Google Cloud Storage bucket as evidence for regulators". ?

File: readme.md [hunk 2]
```diff
@@ -1,25 +1,84 @@
...
+          tests-folder: web/test-results
+          gcp-bucket-name: count-dracula-continous-compliance-prod
+          gcp-application-credentials: ${{ secrets.GOOGLE_APPLICATION_CREDENTIALS }}
+          docs-folder: web/docs
+          github-token: ${{ secrets.GITHUB_TOKEN }}
 ```
 
+## Configuration
+
+The `with` portion of the workflow must be configured before the action will work. You can add these in the with section found in the examples above. Any `secrets` must be referenced using the bracket syntax and stored in the GitHub repository's `Settings/Secrets` menu. You can learn more about setting environment variables with GitHub actions here.
```
Comment (DimplePatel) [inline readme.md:61]: is here supposed to be a hyperlink?

File: readme.md (hunk 1 above)
Comment (pataruco) [reply readme.md:9]: Thanks

File: readme.md (hunk 2 above)
Comment (pataruco) [reply readme.md:61]: not really

File: src/steps/readme.md [hunk 3]
```diff
@@ -7,5 +7,6 @@ This GitHub action is following these steps in secuencial order:
 1. Create a compliance folder on the root of the repo
```
Comment (DimplePatel) [inline src/steps/readme.md:7]: I just noticed there's a typo on line 5 (before the bit you just wrote): 
It should say sequential rather than secuencial

**Review by DimplePatel — CHANGES_REQUESTED** [review]

File: src/steps/readme.md [hunk 4]
```diff
@@ -7,5 +7,6 @@ This GitHub action is following these steps in secuencial order:
...
 2. If `tests-folders` input is given it would copy the contents of the folder and paste it into compliance folder
 3. If `docs-folders` input is given it would copy the contents of the folder and paste it into compliance folder
-4. It would compress (zip) compliance folder and the zip file with an Unix timestamp, e.g. `1617971697.zip`
-5. It would upload the zip file to a Google Cloud storage bucket
+4. If `github-token` input is given it would gather information about the current pull request and commits associated with it and and paste it into compliance folder
```
Comment (DimplePatel) [inline src/steps/readme.md:10]: You have a double 'and' on this sentence, only need one :)

File: src/steps/readme.md (hunk 3 above)
Comment (pataruco) [reply src/steps/readme.md:7]: Thanks

File: src/steps/readme.md (hunk 4 above)
Comment (pataruco) [reply src/steps/readme.md:10]: Thanks

**Review by DimplePatel — APPROVED** [review]

**Pull request merged**